public/locales/en/translation.json

```diff
@@ -103,7 +103,8 @@
             "uninitialized": "The scenario has not been run yet"
           },
           "button": {
-            "downloadlogs": "Download logs"
+            "downloadlogs": "Download logs",
+            "refresh": "Refresh"
```

Comment:
Since we can not add a tooltip when the button is disabled, display "Refresh (will then be disabled for 15 seconds)"